AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -94,6 +94,8 @@ expressen.se##.pa-dialog
 italianfoodlovers.de##.lightbox-backdrop
 palit.com###cookieok
 automation.siemens.com###cm-banner
+spidersweb.pl#$#.fc-consent-root { display: none !important; }
```

Comment:
Are you sure? I see `.fc-ab-root`, which can be added to group of `#$#body div.fc-ab-root { display: none !important; }`